### DISCUSSION TITLE
Came across a new prisma tool being developed

### DISCUSSION THREAD

**Discussion by BossDaily** [prisma/prisma#18299] [Q&A] [answered]
Im just going to shill this project (I didn't make this nor do I have any involvement in its development)

<div align="center">
<a href="https://turboprisma.js.org">
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://turboprisma.js.org/apple-touch-icon.png">
      <img src="https://turboprisma.js.org/logo-white-theme.png" height="128">
    </picture>
    <h1 align="center">Turboprisma</h1>
  </a>
  <a href="https://www.npmjs.com/package/turboprisma"><img src="https://img.shields.io/npm/dt/turboprisma" /></a>
  <a href="https://github.com/DavidHancu/turboprisma/blob/main/CONTRIBUTING.MD"><img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" /></a>
  <a href="https://github.com/DavidHancu/turboprisma/blob/main/LICENSE"><img src="https://img.shields.io/github/license/DavidHancu/turboprisma" /></a>
  <br />
  <br />
  <a href="#what-is-turboprisma">What is Turboprisma?</a>
  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
  <a href="#links">Links</a>
  
  <br />
  <hr />
</div>

## What is Turboprisma?

Turboprisma is a fast and performant Prisma runtime that provides an unified access framework, abstractions and new features on top of the Prisma schema, CLI and tools. It is fully developed in Rust, allowing for extreme speeds and optimizations which are just not possible with Prisma.

## Links

Check out our API Documentation - [API Documentation](https://turboprisma.js.org/docs)

Get Started with Turboprisma - [Getting Started](https://turboprisma.js.org/docs/getting-started)

**Comment by nurul3101** [comment, marked as answer]
Have you tried out Turboprisma? How was your experience?

**Reply by nikolasburk** [reply]
Shoutout to @davidhancu for his work on this 🎉
